### PR TITLE
Operations: add logging instructions with Fluentbit

### PR DESCRIPTION
**Pull request by heitorPB**
Online: https://omnivector-solutions.github.io/osd-documentation/testing/operations/logging.html

### CODE REVIEW

**Review by jamesbeedy — APPROVED** [review]
👍

File: operations/logging.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
+.. _logging:
```
Comment (dusktreader) [inline operations/logging.rst:1]: Out of curiosity only: what does this do?

**Review by dusktreader — APPROVED** [review]
Nice!

File: operations/logging.rst (hunk 1 above)
Comment (heitorPB) [reply operations/logging.rst:1]: Creates a Sphinx label. Anywhere in the docs we can `:ref:'logging´` and Sphinx will know it refers to this file, wherever it is located. See https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role

**Pull request merged**